styles.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+    font-family: "Figtree-ExtraBold";
+    src: url('./assets/fonts/static/Figtree-ExtraBold.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: "Figtree-Medium";
+    src: url('./assets/fonts/static/Figtree-Medium.ttf') format('truetype');
+}
+
+body {
+    background-color: rgb(244, 208, 78);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+img {
+    border-radius: 10px;
+}
+
+h1, h2, p {
+    line-height: 150%;
+    letter-spacing: 0px;
+}
+
+h1 {
+    font-size: 24px;
```

Comment:
For typography attributes such as `font-size`, it's better to use `rem` rather than `px` for accessibility reasons. `rem` will scale to match users' font size preferences (e.g. someone who has trouble seeing will set their browser to use larger font sizes)

`px` still has its place; it depends on the design whether to use `px` or `rem` for things like padding and margin, for example